### PR TITLE
Updating our old friend Jackson & Jackson Databind

### PR DESCRIPTION
**Pull request by paulbrown1982**
As reported by Snyk: https://app.snyk.io/org/the-guardian-cuu/project/923895c3-3d65-4140-a01a-f1f5f1271fc0/

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 language: scala
 sudo: false
 jdk:
-- oraclejdk8
+- oraclejdk9
```
Comment (rtyley) [inline .travis.yml]: Ah, we don't want to use the Oracle JDK at all, due to the Oracle licensing trap: https://blog.joda.org/2018/09/do-not-fall-into-oracles-java-11-trap.html

See also https://github.com/guardian/play-googleauth/pull/66

File: build.sbt [hunk 2]
```diff
@@ -11,8 +11,8 @@ resolvers += "scalaz-bintray" at "https://dl.bintray.com/scalaz/releases"
 scalaVersion := "2.12.6"
 scalacOptions ++= List("-feature", "-deprecation")
 
-val jacksonVersion = "2.9.9"
-val jacksonVersionBump = "2.9.9.1"
+val jacksonVersion = "2.9.10"
+val jacksonVersionBump = "2.9.10.1"
```
Comment (rtyley) [inline build.sbt]: Maybe go even higher?! See https://github.com/guardian/ophan/pull/3476/files

Comment (paulbrown1982) [reply build.sbt]: This branch build didn't deploy so I'll revise things.

Comment (paulbrown1982) [reply build.sbt]: It deploys using this so I'll go this far and if Snyk is still complaining I'll raise another PR.

**Pull request merged**